can/map/map.js

```diff
@@ -523,7 +523,7 @@ var mapOverwrites = {	// ## can.Model#bind and can.Model#unbind
 	___set: function (base, connection) {
 		return function(prop, val){
 			base.apply(this, arguments);
-			if ( prop === connection.idProp && this._bindings ) {
```

Comment:
In this case I think it's better to preserve a leading underscore, as this property now belongs to a map where any event name could be added by users (and most of those will likely be named after definemap properties)